screenpipe-app-tauri/lib/hooks/use-settings.tsx

```diff
@@ -159,93 +163,77 @@ const DEFAULT_IGNORED_WINDOWS_PER_OS: Record<string, string[]> = {
-  }, [settings]);
-
-  const resetSetting = async (key: keyof Settings) => {
-    if (!store) {
-      await initStore();
-    }
-
-    const defaultSettings = createDefaultSettingsObject(platform());
-
-    try {
-      const updatedSettings = { ...settings, [key]: defaultSettings[key] };
-      setSettings(updatedSettings);
-      await store!.set(key, defaultSettings[key]);
-      // No need to call save() as we're using autoSave: true
-    } catch (error) {
-      console.error(`failed to reset setting ${key}:`, error);
-      // revert local state if store update fails
-      setSettings(settings);
-    }
-  };
-
-  const resetSettings = async () => {
-    const userSettings = createDefaultSettingsObject(platform());
+// Model definition
+export interface StoreModel {
+  settings: Settings;
+  setSettings: Action<StoreModel, Partial<Settings>>;
+  resetSettings: Action<StoreModel>;
+  resetSetting: Action<StoreModel, keyof Settings>;
+}
 
-    updateSettings(userSettings);
-  };
+function createDefaultSettingsObject(currentPlatform: string): Settings {
+  let defaultSettings = { ...DEFAULT_SETTINGS };
 
-  const loadSettings = async () => {
-    if (!store) {
-      await initStore();
-    }
+  const ocrModel =
+    currentPlatform === "macos"
+      ? "apple-native"
+      : currentPlatform === "windows"
+      ? "windows-native"
+      : "tesseract";
 
-    try {
-      const currentPlatform = platform();
-      const userSettings = await createUserSettings(store!, currentPlatform);
+  defaultSettings.ocrEngine = ocrModel;
+  defaultSettings.fps = currentPlatform === "macos" ? 0.2 : 1;
+  defaultSettings.platform = currentPlatform;
 
-      setSettings({ ...userSettings, isLoading: false });
-    } catch (error) {
-      console.error("failed to load settings:", error);
-      setSettings((prevSettings) => ({ ...prevSettings, isLoading: false }));
-    }
-  };
+  defaultSettings.ignoredWindows = [
+    ...DEFAULT_IGNORED_WINDOWS_IN_ALL_OS,
+    ...(DEFAULT_IGNORED_WINDOWS_PER_OS[currentPlatform] ?? []),
+  ];
 
-  useEffect(() => {
-    loadSettings();
-  }, []);
+  return defaultSettings;
+}
 
-  const updateSettings = async (newSettings: Partial<Settings>) => {
-    if (!store) {
-      await initStore();
+// Export the store directly
+export const store = createStoreEasyPeasy<StoreModel>(
+  persist(
+    {
+      settings: DEFAULT_SETTINGS as Settings, // Start empty, will be populated in initStore
+      setSettings: action((state, payload) => {
+        state.settings = {
+          ...state.settings,
+          ...payload,
+        };
+      }),
+      resetSettings: action((state) => {
+        state.settings = createDefaultSettingsObject(state.settings.platform);
+      }),
+      resetSetting: action((state, key) => {
+        const defaultValue = createDefaultSettingsObject(state.settings.platform)[key];
+        (state.settings as any)[key] = defaultValue;
+      }),
+    },
+    {
+      storage: 'localStorage',
```

Comment:
users will lose all their settings 

and `screenpipe-js` will not work anymore

not big fan of local storage - it should be on disk